cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,21 +1,33 @@
+import {
+  ItemPage, LoginPage, ProductListPage,
+  ShoppingCartPage, CheckoutCompletePage,
+  InformationPage, OverviewPage
+} from '../page'
+
+const loginPage = new LoginPage()
+const productList = new ProductListPage()
+const shoppingCart = new ShoppingCartPage()
+const itemPage = new ItemPage()
+const verifyOrderCompletion = new CheckoutCompletePage()
+const informationPage = new InformationPage()
+const overviewe = new OverviewPage()
+const userName = 'standard_user'
+const pass = 'secret_sauce'
+const firstName = 'Cypress'
+const lastName = 'Workshop'
+const postalCode = '00000'
+const messageValidateOrderComplete = 'Thank you for your order!'
+
 describe('Buy a black t-shirt', () => {
   it('then the t-shirt should be bought', () => {
-    cy.visit('https://www.saucedemo.com/')
-    cy.get('.login-box > form > div > input#user-name').type('standard_user')
-    cy.get('.login-box > form > div > input#password').type('secret_sauce')
-    cy.get("input[type='submit']").click()
-    cy.get('#item_1_img_link').click()
-    cy.get('#add-to-cart-sauce-labs-bolt-t-shirt').click()
-    cy.get('#shopping_cart_container > a').click()
-    cy.get('#checkout').click()
-    cy.get('#first-name').type('Cypress')
-    cy.get('#last-name').type('Workshop')
-    cy.get('#postal-code').type('00000')
-    cy.get('#continue').click()
-    cy.get('#finish').click()
-    cy.get('#contents_wrapper > .checkout_complete_container > h2').should(
-      'have.text',
-      'Thank you for your order!'
-    )
+    loginPage.visitLoginPage()
+    loginPage.signIn(userName, pass)
+    productList.selectTShirt()
+    itemPage.addTShirtItem()
+    shoppingCart.goShoppingCart()
+    shoppingCart.checkoutCart()
+    informationPage.checkoutUserInformation(firstName, lastName, postalCode)
+    overviewe.finishOverview()
+    verifyOrderCompletion.checkoutOrdenComplete(messageValidateOrderComplete)
```

Comment:
in this case what you should change is the method's name, the previous name for the page object was ok. Sorry for not be clear. Suggestion: `checkout.verifyOrderCompletion(messageValidateOrderComplete)`

Reply:
fix suggestion, thanks